webapp/security/helpers.py

```diff
@@ -1,105 +1,104 @@
+            if versions:
+                if status["release_codename"] not in versions:
+                    package_count -= 1
+                    continue
 
-        # If any package status is released,
-        # count how many packages have that status
-        elif any(d["status"] == "released" for d in package["statuses"]):
-            if more_packages:
-                total_fixed = 0
-                total_fixable = 0
-                for package in cve["packages"]:
-                    for status in package["statuses"]:
-                        if status["status"] == "released":
-                            total_fixed += 1
-                        elif status["status"] not in {
-                            "DNE",
-                            "not-affected",
-                        }:
-                            total_fixable += 1
+            if status["release_codename"] != "upstream":
+                if status["status"] == "ignored":
+                    if any(
+                        indicator in description
+                        for indicator in ignored_low_indicators
+                    ):
+                        status_count["ignored-low"] += 1
+                    else:
+                        status_count["ignored-high"] += 1
+                elif status["status"] == "released":
+                    status_count["released"] += 1
+                elif status["status"] in vulnerable_indicators:
+                    status_count["vulnerable"] += 1
+                elif status["status"] == "needs-triage":
+                    status_count["needs-triage"] += 1
+                elif status["status"] == "not-affected":
+                    status_count["not-affected"] += 1
+                elif status["status"] == "DNE":
+                    status_count["DNE"] += 1
             else:
-                fixed_count = Counter(
-                    d["status"]
-                    for d in package["statuses"]
-                    if d["status"] == "released"
-                )
-                total_fixed = fixed_count["released"]
-                total_fixable = len(package["statuses"])
+                # Exclude upstream release from package count
+                package_count -= 1
 
-            cve["summarized_status"] = {
-                "name": "Some fixed",
-                "fixed_count": total_fixed,
-                "total_count": total_fixable,
-            }
+    # Check if all statuses are the same
+    count = 0
+    for key, value in status_count.items():
+        if key != "DNE" and value > 0:
+            count += 1
+            key_with_non_zero_value = key
 
-        elif any(
-            d["status"] in vulnerable_indicators for d in package["statuses"]
-        ):
-            cve["summarized_status"] = {
-                "name": "Vulnerable",
-            }
+    if count == 1:
+        if key_with_non_zero_value == "ignored-high":
+            cve["summarized_status"] = friendly_names["ignored"]
```

Comment:
As per the comment in `views.py`, we can return the value of the status rather than setting it such as:
```suggestion
            return friendly_names["ignored"]
```

If there is a specific reason that you'd prefer to set it within the function please let me know, perhaps we could change the function name to `set_summarized_status` instead if that's the case?

Reply:
No specific reason, update pushed :)